### PR TITLE
Add use_your_own_data example and fix HGT example inconsistency

### PR DESCRIPTION
**Pull request by zhjwy9343**
*Issue #, if available:*

*Description of changes:*
This PR includes two parts:

1. Add a new folder, named "examples/use_your_own_data/" and add a yaml file that can use ACM data as an example of customer data and train on built-in RGCN model.
2. Modify the HGT model's README to fit the OSS structure, move acm_data.py to the examples/ folder.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: examples/acm_data.py [hunk 1]
```diff
@@ -80,11 +87,14 @@ def create_acm_graph(dowload_path='/tmp/ACM.mat', output_path=None):
     
     parser.add_argument('--download-path', type=str, default='/tmp/ACM.mat',
                         help="The path of folder to store downloaded ACM raw data")
+    parser.add_argument('--dataset-name', type=str, default='acm',
```
Comment (classicsong) [inline examples/acm_data.py:90]: Are you going to support other datasets?

Comment (classicsong) [reply examples/acm_data.py:90]: The name of the example is already acm_data.py I don't think we need this argument.

**Review by classicsong — APPROVED** [review]
LGTM. Left one comment.

File: examples/acm_data.py (hunk 1 above)
Comment (zhjwy9343) [reply examples/acm_data.py:90]: I provide this argument to let users know that they can change the name.

Comment (classicsong) [reply examples/acm_data.py:90]: If they change the name, it is still acm, right?

Comment (zhjwy9343) [reply examples/acm_data.py:90]: yes, the data is still acm data, but the saved file will be <theirname>.dgl. And in the rest steps they need to use their name to replace "acm" in all scripts.

**Pull request merged**